### DISCUSSION TITLE
Votación para la traducción de la palabra en inglés "root"

### DISCUSSION THREAD

**Discussion by NGHdeveloper** [reactjs/es.reactjs.org#844] [Polls]
La palabra de origen inglés _"root"_ ¿deberíamos traducirla al español con su equivalente "raíz"?, o dejarla como _"root"_ tal como como React la define para su elemento _root_ en [Tu primer componente](https://es.react.dev/learn/your-first-component#components-all-the-way-down). Yo particularmente prefiero la opción de dejarla tal como React la define, para una mejor comprensión de tutoriales y referencias en funciones, etc. Otra opción sería dejarla como anglicismo de esta manera, _root_ (raíz), además de agregarla al glosario.

**Comment by Byron2016** [comment]
Tomar en cuenta que lo decidido no consta ni en el [Glosario](https://github.com/reactjs/es.react.dev/wiki/Glosario) ni en [Guia de estilo](https://github.com/reactjs/es.react.dev/wiki/Gu%C3%ADa-de-estilo), habría que especificarse en algún lugar.

**Comment by luquefv** [comment]
este programa es muy dificil, no entiendo nada
